City-of-Pori/pori-kada
# PORI-342 Content based on user domain

To test:

1. drush fra -y && drush cc all
2. First log in with your user and edit the profile https://local.pori.fi/en/user/1/edit
3. Check only either one, Pori.fi domain or Visitpori.fi domain for the Domain access box.
4. Go to https://local.pori.fi/en/admin/content and make sure only the content that is attached to the domain that you chose to your profile is visible. There should be "Domain" filter on the top of the view where you can change the value but by default it should have the same value that you have in your profile.
5. Add some images to scald with the profile you are currently logged in.
6. When having Pori.fi domain chosen in the profile, go to https://local.pori.fi/en/admin/content/atoms and make sure you can see the images that you added.
7. Now log in to local.visitpori.fi and go to https://local.visitpori.fi/en/admin/content/atoms. You shouldn't be able to see the images that you added on the listing.
8. Go to edit any page-content type node in local.visitpori.fi and click on the scald tool on the right. You shouldn't see any of the images that you added there either.
9. Now go back to local.pori.fi and edit any page-content type node and click on the scald tool on the right. You should now see all the images that you added earlier.
10. Go to switch the domain on your profile to visitpori.fi and you should see exactly the opposite.
